packages/discovery-provider/plugins/pedalboard/apps/relay/src/middleware/logging.ts

```diff
@@ -30,7 +30,7 @@ export const outgoingLog = (request: Request, response: Response) => {
   const path: string = route.path
   if (!path.includes("health")) {
     logger.info(
-      { route, method, ctx, responseTime, statusCode },
+      { route, method, abi: ctx.ctx.validatedRelayRequest.encodedABI, responseTime, statusCode },
```

Comment:
putting too much info in logs, this just cuts down to abi so we can decode it easily